src/consts.rs

```diff
@@ -9,5 +9,9 @@ pub const PROTOCOL_FEES_PERCENTAGE: u8 = 10;
 /// The discriminator for the external undelegate instruction.
 pub const EXTERNAL_UNDELEGATE_DISCRIMINATOR: [u8; 8] = [196, 28, 41, 206, 48, 37, 51, 167];
 
+/// The discriminator for the external hook after finalization is complete
+/// For anchor: corresponds to function/instruction name delegation_program_call_handler
+pub const EXTERNAL_CALL_HANDLER_DISCRIMINATOR: [u8; 8] = [157, 197, 228, 30, 0, 80, 121, 135];
```

Comment:
in a second iteration, the discriminator of call_handler could be user defined (allowing support of multiple handler per program)

Reply:
Agreed